playbooks/openshift-master/private/tasks/enable_bootstrap.yml

```diff
@@ -0,0 +1,23 @@
+---
+- when: openshift_master_bootstrap_enabled | default(false) | bool
+  block:
+  - name: Setup the master bootstrap settings
+    import_role:
+      name: openshift_master
+      tasks_from: bootstrap_settings.yml
+
+  - name: Setup the bootstrap kubeconfig
+    import_role:
+      name: openshift_master
+      tasks_from: bootstrap.yml
+
+  - name: Setup the node group config maps
+    import_role:
+      name: openshift_node_group
+    run_once: True
+
+  - name: Setup the node bootstrap auto approver
+    import_role:
+      name: openshift_bootstrap_autoapprover
+    when:
```

Comment:
@kwoodson is this a typo and as such should be removed ?

Reply:
Thanks, updated.